src/Text/__tests__/index.test.js

```diff
@@ -2,15 +2,34 @@
 import * as React from "react";
 import { shallow } from "enzyme";
 
-import Text from "../";
+import Text from "../index";
+import ThemeProvider from "../../Theming/ThemeProvider";
 
-describe("Rendered with children", () => {
-  it("Should contain text", () => {
+describe("Text", () => {
+  it("should contain children", () => {
+    const children = "Children text";
+    const component = shallow(
+      <ThemeProvider>
+        <Text type="primary" size="small">
+          {children}
+        </Text>
+      </ThemeProvider>,
+    );
+    expect(
+      component
+        .children()
+        .children()
+        .exists(),
+    ).toBe(true);
```

Comment:
Pls check that it contains `Children text`

Reply:
Fixed.